skynet/constants.py

```diff
@@ -1,113 +1,115 @@
-#!/usr/bin/python
 import msgspec
+
+from enum import Enum
 from typing import Literal
 
 VERSION = '0.1a12'
 
 DOCKER_RUNTIME_CUDA = 'skynet:runtime-cuda'
 
 
-class Size(msgspec.Struct):
-    w: int
-    h: int
-
-
 class ModelDesc(msgspec.Struct):
-    short: str
-    mem: float
-    size: Size
-    tags: list[Literal['txt2img', 'img2img', 'inpaint']]
+    short: str  # short unique name
```

Comment:
also maybe in future we'll need a `.key/name: str` which also holds the same key as in the `MODELS: dict[str, ModelDesc]` so that if there's ever a case where you have the descr ref but don't know the name it came from?

def can be added later.

Reply:
yeah agreed